### PR TITLE
Log traceback when watcher fails

### PR DESCRIPTION
**Pull request by salv-orlando**
A full traceback could be quite useful for debugging and troubleshooting.
This patch add an additional vlog.exception statement, preserving the
original vlog.warn statement, which has been only slightly changed to
enclose the exception reason in double quotes

Signed-off-by: Salvatore Orlando <salv.orlando@gmail.com>

### CODE REVIEW

File: ovn_k8s/watcher/pod_watcher.py [hunk 1]
```diff
@@ -72,7 +72,6 @@ def _process_pod_event(self, event):
             has_conn_event = True
         else:
             return
-
```
Comment (shettyg) [inline ovn_k8s/watcher/pod_watcher.py]: Unrelated change?

**Comment by salv-orlando** [comment]
The failure was due to error that made into master because of PR #20 - I just need to rebase this

File: ovn_k8s/watcher/pod_watcher.py (hunk 1 above)
Comment (salv-orlando) [inline ovn_k8s/watcher/pod_watcher.py]: yes will fix

**Comment by shettyg** [comment]
LGTM
